### PR TITLE
add method to mint kyc + test

### PR DESCRIPTION
**Pull request by sczembor**
+ add `admin_mint_kyc` method
+ unit tests
+ integration test

### CODE REVIEW

File: contracts/oracle/src/lib.rs [hunk 1]
```diff
@@ -333,6 +333,68 @@ impl Contract {
...
+    #[private]
+    pub fn admin_mint_kyc_callback(
+        &mut self,
+        #[callback_result] last_result: Result<Vec<TokenId>, PromiseError>,
+    ) -> CallbackResult<Vec<TokenId>, &str> {
+        match last_result {
+            Ok(v) => CallbackResult::Ok(v),
+            Err(_) => CallbackResult::Err("registry.sbt_mint failed"),
+        }
+    }
```
Comment (robert-zaremba) [inline contracts/oracle/src/lib.rs:396]: callback is not needed

File: contracts/oracle/src/lib.rs [hunk 2]
```diff
@@ -333,6 +333,68 @@ impl Contract {
...
+        require!(
+            env::attached_deposit() >= storage_deposit,
+            format!(
+                "Requires attached deposit at least {} yoctoNEAR",
+                storage_deposit
+            )
+        );
+
+        let now: u64 = env::block_timestamp_ms();
+        let mut tokens_metadata: Vec<(AccountId, Vec<TokenMetadata>)> = Vec::new();
```
Comment (robert-zaremba) [inline contracts/oracle/src/lib.rs:356]: we can create a vector with predetermined capacity

File: contracts/oracle/src/lib.rs [hunk 3]
```diff
@@ -333,6 +333,68 @@ impl Contract {
         Ok(())
     }
 
+    #[payable]
+    #[handle_result]
+    pub fn admin_mint_kyc(
```
[the remark this answers is not in the document]
Comment (sczembor) [reply contracts/oracle/src/lib.rs:340]: done

File: contracts/oracle/src/lib.rs (hunk 1 above)
Comment (sczembor) [reply contracts/oracle/src/lib.rs:381]: done

File: contracts/oracle/src/lib.rs (hunk 2 above)
Comment (sczembor) [reply contracts/oracle/src/lib.rs]: done

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**